mantidproject/mantid
# Prevent units that are not suitable from being used in convertunits

**Description of work.**
Ensures that the target unit validator only lists "convertible units


**To test:**
1. Check that the "fake" units of Empty, Label, Degrees and Temperature are not available in the drop down for ConvertUnits
1. Check that the documentation does not list them as valid options.

Fixes #27458. 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
